### ISSUE TITLE
Add a Theme  Mode Import/Export Tab or Section

### ISSUE THREAD

**Issue by shofer4eto** [valendesigns/option-tree#478]
Hi,

I would love to see a way to include Options Export/Import into a OT section as when running in Theme Mode that would be the most appropriate way from a UI perspective. I am currently hiding the settings/documentation pages. Can I also filter only the Options and remove the Settings and Layouts. Thanks a lot in advance! Great work! :)

**Comment by vburlak** [comment]
@shofer4eto I think you're looking for #259

**Comment by shofer4eto** [comment]
Hey,

Not really, I found that post earlier too and what I am after is what "guldoe" suggested on Nov 11, 2014 ... Unfortunately Derek recommended that it is a faulty implementation and addded the feature to be included in 2.5. That's why I need to know if it has been implemented yet (the separation of Import/Export and the Theme Options, still being able to load the Import/Export section at the Theme Options level). Any clues/suggestions are welcome. Thanks!
